filesystem/node.go

```diff
@@ -32,9 +32,12 @@ var _ = fs.Node(&Node{})
 // Lookup
 var _ = fs.NodeStringLookuper(&Node{})
 
+// HandleReadDirAller
+var _ = fs.HandleReadDirAller(&Node{})
+
 // Node implements the Node interface.
 type Node struct {
```

Comment:
I'm not certain about that. If we use Node to implement both Handler and Node interfaces, we should be fine. still TBD.